tests/test_origin_indexing.py

```diff
@@ -34,28 +34,35 @@ def test_origins_dict(origins_dict):
     assert len(origins_dict) == origins_dict[-1]["index"] + 1
 
 
-def test_content_dict(content_dict):
+def test_config(config):
     batch_id = "content.blocking_blocked-{index}"
-    assert content_dict[batch_id.format(index=0)] == 2046
-    assert content_dict[batch_id.format(index=1)] == 441
+    part_0 = [d["n_data"] for d in config if d["batch_id"] == batch_id.format(index=0)][
+        0
+    ]
+    part_1 = [d["n_data"] for d in config if d["batch_id"] == batch_id.format(index=1)][
+        0
+    ]
```

Comment:
Maybe this would be shorter?

```suggestion
    (part_0, part_1) = [[d["n_data"] for d in config if d["batch_id"] == batch_id.format(index=i)][i] for i in (1, 2)]
```